src/builtin.c

```diff
@@ -209,18 +209,23 @@ builtinLoad(struct VM *vm) {
     // TODO: exception?
     assert("wrong path");
   }
+
+  vm->gcSave = vm->pcData;
```

Comment:
Have to save the `InstrPrimitive` instruction, there is no reference to it from `vm`, but when `load` exit, it's used again!!!